R/update_package_object.R

```diff
@@ -0,0 +1,49 @@
+#' update_package_object
```

Comment:
Probably want to make this a sentence.  Something like: Update a data object and accompanying package resource map.